BLocal.Web.Manager.MVC3/Views/Home/Index.cshtml

```diff
@@ -17,7 +17,7 @@
                 if (data === "ok") {
                     location.href = "/Home/Overview";
                 } else if (data === "bad_domain") {
-                    alert("Your google account's domain is not allowed to login using google. You will be logged out from current account. Please try to log in using good account.");
+                    alert("Your google account's domain is not allowed to login using google. You will be logged out from current account. Please try to log in using account allowed to enter.");
```

Comment:
Please revise the statement - it should state somehting like Your xxx is not allowed to use this service.